doc/dox_comments/header_files/ssl.h

```diff
@@ -7604,6 +7755,27 @@ WOLFSSL_API int   wolfSSL_DTLS_SetCookieSecret(WOLFSSL*,
                                                const unsigned char*,
                                                unsigned int);
 
+/*!
+    \brief This function retrieves the Device Id.
+
+    \return rng upon success.
+    \return NULL if ssl is NULL.
+
+    \param ssl pointer to a SSL object, created with wolfSSL_CTX_new().
```

Comment:
"ssl" should be created with "wolfSSL_new()".  Only contexts (WOLFSSL_CTX) are created with wolfSSL_CTX_new().